assets/airports/eddf.json

```diff
@@ -6,940 +6,3986 @@
+            },
+            "body": [],
+            "exitPoints": {
+                "ANEKI": ["ANEKI"]
+            },
+            "draw": [
+                ["DF152", "DF150", "DF157", "ANEKI"],
+                ["DF144", "DF154", "DF160", "RID"],
+                ["DF135", "DF142", "DF143", "DF137", "RID"],
+                ["DF134", "DF141", "DF143"],
+                ["RID", "ANEKI"]
+            ]
+        },
+        "DKB": {
+            "icao": "DKB",
+            "name": "Dinkelsbühl",
+            "suffix": {"07C":"1D", "07R":"1D", "18":"6L", "25L":"8F", "25C":"8F"},
             "rwy": {
-                "25C": ["DF234", ["DF170", "S185-"], ["DF172", "A25+"], "PABVI", "SIVDO", "KUPIP", "MASIR"],
-                "25L": ["DF235", ["DF171", "S185-"], ["DF172", "A25+"], "PABVI", "SIVDO", "KUPIP", "MASIR"],
-                "07C": ["DF139", ["DF146", "A20+"], "ODAGA", "KUSOM", "GUBAX"],
-                "07R": ["DF140", ["DF146", "A20+"], "ODAGA", "KUSOM", "GUBAX"],
-                "18":  ["DF197", ["DF156", "S220-"], "DF164", "KUPIP", "MASIR"]
+                "07C": [["DF152", "A08+"], ["DF150", "S220-"], "DF153"],
+                "07R": [["DF152", "A08+"], ["DF150", "S220-"], "DF153"],
+                "18":  [["RID", "A08+|S220-"]],
+                "25L": [["^DF135", "A08+"], "DF142", "DF143", ["DF137", "S210-"], ["DF159", "A25+"]],
+                "25C": [["^DF134", "A08+"], "DF141", "DF143", ["DF137", "S210-"], ["DF159", "A25+"]]
             },
-            "body": ["RAVKI", "DITAM", "OBOKA"],
+            "body": ["AMTIX", "AKONI"],
             "exitPoints": {
-                "BIBTI": ["BIBTI"]
+                "DKB": ["DKB"]
             },
-            "draw": [["DF172", "PABVI", "SIVDO", "KUPIP", "MASIR", "RAVKI"], ["DF197", "DF156", "DF164", "KUPIP"], ["DF235", "DF171"], ["DF140", "DF146"], ["DF146", "ODAGA", "KUSOM", "GUBAX", "RAVKI", "DITAM", "OBOKA", "BIBTI"]]
+            "draw": [
+                ["DF152", "DF150", "DF153", "AMTIX"],
+                ["RID", "AMTIX"],
+                ["DF135", "DF142", "DF143", "DF137", "DF159", "AMTIX"],
+                ["DF134", "DF141", "DF143"],
+                ["AMTIX", "AKONI", "DKB"]
+            ]
+        },
+        "KOMIB": {
+            "icao": "KOMIB",
+            "name": "Komib",
+            "suffix": {"07C":"3D", "07R":"3D"},
+            "rwy": {
+                "07C": [],
+                "07R": []
+            },
+            "body": [["DF149", "A08+|S220-"], "DF151", "DF169", "AGOLO", "OKTUM"],
+            "exitPoints": {
+                "KOMIB": ["KOMIB"]
+            },
+            "draw": [
+                ["DF149", "DF151", "DF169", "AGOLO", "OKTUM", "KOMIB"]
+            ]
         },
         "MARUN": {
             "icao": "MARUN",
             "name": "Marun",
-            "suffix": {"25C":"2M", "25L":"2M", "07C":"4E", "07R":"4E", "18":"4S"},
+            "suffix": {"07C":"9D", "07R":"9D", "18":"4T", "25L":"6G", "25C":"6G"},
```

Comment:
They normally use the 1K and 1W departures out of 25 :slightly_frowning_face:

Reply:
Oh ok, I couldn't find the fix coordinates from the German AIP but will look harder again